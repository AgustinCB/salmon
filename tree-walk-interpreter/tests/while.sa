var counter = 0;

while (counter < 5) {
    print "HOLA";
    counter = counter + 1;
}
print "CHAU";