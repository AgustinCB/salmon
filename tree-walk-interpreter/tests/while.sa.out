HOLA
HOLA
HOLA
HOLA
HOLA
CHAU
